src/components/Shared/DecisionResult.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import LegislationAdoptedIcon from "../Cards/LegislationCard/LegislationAdoptedIcon";
+import LegislationRejectedIcon from "../Cards/LegislationCard/LegislationRejectedIcon";
+import LegislationInProgressIcon from "../Cards/LegislationCard/LegislationInProgressIcon";
+import { MATTER_STATUS_DECISION } from "../../constants/ProjectConstants";
+import { VOTE_DECISION } from "../../constants/ProjectConstants";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+interface DecisionResultProps {
+  /**Result of the vote or council decision.  This is an enum value, you can see the enumeration in ProjectConstants */
+  result: string;
```

Comment:
You can use the enum itself as the type here.

`result: VOTE_DECISION;`

Reply:
I wanted this to be a reusable component shared between individual vote decisions and council-body decisions.  Is there a typescript-y way of making something _either_ a VOTE_DECISION _or_ a MATTER_STATUS_DECISION?  Normally this is where `interface` would come in handy in Swift: both enums would conform to a broader type.  

fake edit:   `result: VOTE_DECISION | MATTER_STATUS_DECISION;` works

Reply:
Oh, nice thought on the reusability. 

Also you got it, the `|` can act as a logic OR for type annotations in TypeScript.